### DISCUSSION TITLE
loadTraits doesn't exists after upgrade from legacy api.

### DISCUSSION THREAD

**Discussion by wyxcoder** [GrapesJS/grapesjs#5180] [Q&A]
After upgrade to new API like below, I got error loadTraits doesn't exist. I wonder with the new API how does it inherit defaultModel?

Components.addType('component-id', {
 model: defaultModel.extend({

to 

Components.addType('component-id', {
 model: {

**Comment by artf** [comment]
Show the complete code please

**Comment by wyxcoder** [comment]
I found example suggest to use this.getTrait().attributes, but it seems I should just use this.attributes
